.github/workflows/build-docker-image.yml

```diff
@@ -32,3 +32,28 @@ jobs:
             with:
               push: true
               tags: ${{ steps.meta.outputs.tags }}
+
+          - name: Install kubectl
```

Comment:
This will be executed on every push to `master` too. Isn't it better to have a new workflow or job that only runs when v-tags are pushed?